src/lib/import-export/tests/export/exporters/default-exporter.test.ts

```diff
@@ -125,13 +129,17 @@ describe( 'DefaultExporter', () => {
 		await exporter.export();
 
 		expect( archiver ).toHaveBeenCalledWith( 'tar', { gzip: true, gzipOptions: { level: 9 } } );
+		expect( getWordPressVersionFromInstallation ).toHaveBeenCalledTimes( 1 );
+		expect( getWordPressVersionFromInstallation ).toHaveBeenCalledWith( '/path/to/site' );
```

Comment:
I wonder if we need to check these expectations. What do you think if instead, we add a test case to check that the file `studio.json` is archived?

Here's a potential approach:
```patch
diff --git forkSrcPrefix/src/lib/import-export/tests/export/exporters/default-exporter.test.ts forkDstPrefix/src/lib/import-export/tests/export/exporters/default-exporter.test.ts
index 87dd21b8bc41287373d5e9909c3bde9c41a805eb..fad98f047c62f58d3cf4d59538dbff39659040fa 100644
--- forkSrcPrefix/src/lib/import-export/tests/export/exporters/default-exporter.test.ts
+++ forkDstPrefix/src/lib/import-export/tests/export/exporters/default-exporter.test.ts
@@ -129,8 +129,6 @@ describe( 'DefaultExporter', () => {
 		await exporter.export();
 
 		expect( archiver ).toHaveBeenCalledWith( 'tar', { gzip: true, gzipOptions: { level: 9 } } );
-		expect( getWordPressVersionFromInstallation ).toHaveBeenCalledTimes( 1 );
-		expect( getWordPressVersionFromInstallation ).toHaveBeenCalledWith( '/path/to/site' );
 	} );
 
 	it( 'should create a zip archive when the backup file ends with .zip', async () => {
@@ -138,8 +136,6 @@ describe( 'DefaultExporter', () => {
 		await exporter.export();
 
 		expect( archiver ).toHaveBeenCalledWith( 'zip', { gzip: false, gzipOptions: undefined } );
-		expect( getWordPressVersionFromInstallation ).toHaveBeenCalledTimes( 1 );
-		expect( getWordPressVersionFromInstallation ).toHaveBeenCalledWith( '/path/to/site' );
 	} );
 
 	it( 'should add wp-config.php to the archive', async () => {
@@ -159,8 +155,17 @@ describe( 'DefaultExporter', () => {
 		expect( mockArchiver.file ).toHaveBeenCalledWith( '/path/to/site/wp-config.php', {
 			name: 'wp-config.php',
 		} );
+	} );
+
+	it( 'should add studio.json to the archive', async () => {
+		const exporter = new DefaultExporter( mockOptions );
+		await exporter.export();
+
 		expect( getWordPressVersionFromInstallation ).toHaveBeenCalledTimes( 1 );
 		expect( getWordPressVersionFromInstallation ).toHaveBeenCalledWith( '/path/to/site' );
+		expect( mockArchiver.file ).toHaveBeenCalledWith( '/tmp/studio_export_123/studio.json', {
+			name: 'studio.json',
+		} );
 	} );
 
 	it( 'should add wp-content files to the archive', async () => {
@@ -195,8 +200,6 @@ describe( 'DefaultExporter', () => {
 			'/path/to/site/wp-content/themes/theme1/index.php',
 			{ name: 'wp-content/themes/theme1/index.php' }
 		);
-		expect( getWordPressVersionFromInstallation ).toHaveBeenCalledTimes( 1 );
-		expect( getWordPressVersionFromInstallation ).toHaveBeenCalledWith( '/path/to/site' );
 	} );
 
 	it( 'should add a database file to the archive when database is included', async () => {
@@ -230,8 +233,6 @@ describe( 'DefaultExporter', () => {
 		await exporter.export();
 
 		expect( mockArchiver.finalize ).toHaveBeenCalled();
-		expect( getWordPressVersionFromInstallation ).toHaveBeenCalledTimes( 1 );
-		expect( getWordPressVersionFromInstallation ).toHaveBeenCalledWith( '/path/to/site' );
 	} );
 
 	it( 'should cleanup temporary files when database is included', async () => {

```

Reply:
Great suggestion!, I added new new specific test here https://github.com/Automattic/studio/pull/430/commits/c1a3dc45be9d179b0f1e3cf237be3701c87064bc